tests/caclmgrd/caclmgrd_bfd_test.py

```diff
@@ -44,6 +44,8 @@ def test_caclmgrd_bfd(self, test_name, test_data, fs):
             call_rc = test_data["call_rc"]
             mocked_subprocess.call.return_value = call_rc
 
+            self.caclmgrd.ControlPlaneAclManager.get_namespace_mgmt_ip = mock.MagicMock()
+            self.caclmgrd.ControlPlaneAclManager.get_namespace_mgmt_ipv6 = mock.MagicMock()
```

Comment:
Why do you add mock? And why can original unit test work without mock?

Reply:
It throws this error: 
```
scripts/caclmgrd:202: in run_commands_pipe
    exitcodes, stdout = getstatusoutput_noshell_pipe(cmd0, cmd1, cmd2, cmd3)
/usr/local/lib/python3.9/dist-packages/sonic_py_common/general.py:52: in getstatusoutput_noshell_pipe
    popens = [Popen(cmd0, stdout=PIPE, universal_newlines=True)]
[OSError]: [Error 9] Bad file descriptor in the fake filesystem: '11'
```
I think it is due to 

> If close_fds is true, all file descriptors except 0, 1 and 2 will be closed before the child process is executed. Otherwise when close_fds is false, file descriptors obey their inheritable flag as described in [Inheritance of File Descriptors](https://docs.python.org/3/library/os.html#fd-inheritance).

in https://docs.python.org/3/library/subprocess.html#popen-constructor
Not sure how original UT works w/o mock

Reply:
You have changed the behavior of this unit test, and we might have better solution.
https://github.com/sonic-net/sonic-host-services/blob/master/tests/caclmgrd/caclmgrd_bfd_test.py#L37
Above line doesn't work with your commit, and I think you should fix this mock and keep the original behavior.

Reply:
I fixed the mock and revert to original behavior. Could you help review again this UT?

Reply:
Other file is still using mock for get_namespace_mgmt_ip and get_namespace_mgmt_ipv6

Reply:
Fixed. And thanks so much for your suggestion!